### ISSUE TITLE
project now is hard to compile

### ISSUE THREAD

**Issue by Dj-jom2x** [Wurst-Imperium/Wurst7#50]
```

> Task :compileJava FAILED
:setting java compiler args
Could not find refmap definition, will be using default name: Wurst-Client-refmap.json
error: cannot access Immutable
  class file for javax.annotation.concurrent.Immutable not found
1 error
``

I fresh clone and gen source

**Comment by Dj-jom2x** [comment]
what exactly java 8 you had.. i even tried the latest jdk-8u241-windows-x64.exe   this calls still missing javax.annotation.concurrent.Immutable not found

fresh clone gen source and gen eclipse and build

**Comment by Alexander01998** [comment]
This has been fixed in c37f445bcdfbbbff1048ed4287e762cc30ec3032.

Turns out BlockPos uses JSR305's `@Immutable` annotation and because Wurst's PathPos extends BlockPos, it now needs that library as well. Strangely this is only a problem when using `build`, not when running the project in Eclipse.

**Comment by Dj-jom2x** [comment]
thank you for fix
